src/app.md

```diff
@@ -234,6 +235,35 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-sizes]: #ex-file-sizes
+<a name="ex-file-sizes"></a>
+## Recursively calculate file sizes at given depth
+
+[min_depth](https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.min_depth) &
```

Comment:
Please prefer the version agnostic links
`https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.min_depth`
instead of fully qualified
`https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.min_depth`

also please split the links into inline:
<pre> The example uses [`WalkDir::min_depth`] and [`WalkDir::max_depth`]to specify depth at which ...</pre>
and footer part (Reference section at the bottom of the basics.md file) 
in the form:
<pre>
[`WalkDir::min_depth`]: https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.min_depth
[`WalkDir::max_depth`]: https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.max_depth
</pre>

This will render the links nicely and will retain readability of the Markdown source code while keeping all the references in single place across examples